priv/cabbage/apps/itest/test/itest/watcher_info_api_test.exs

```diff
@@ -26,9 +26,10 @@ defmodule WatcherInfoApiTest do
   @to_milliseconds 1000
```

Comment:
can you rename this test and it's feature file?

Reply:
done